http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20240605_add_update_app_log_table.xml

```diff
@@ -0,0 +1,33 @@
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="jcanas" id="add_update_app_log_table">
+
+        <createTable tableName="UPDATE_APP_LOG">
```

Comment:
There's an AppError object but I just realized that there's not a direct link between the two records; we might want to add a foreign key to one of these tables

Reply:
yeah, we already have an `APP_ERROR` table to store this information we a reason already corresponding to updates. I think it is much better to do a join on this table rather than duplicate this information in the database

Reply:
Definitely, I just do not see how to easily do that join for now, because appID alone is not enough right?

Reply:
I am fine either way, if you can modify the design doc accordingly so users know what to do we should be good 👍

Reply:
it should be, the appId in the new table matches the field of the same name as FK references to the APP table